elasticjob-error-handler/elasticjob-error-handler-type/elasticjob-error-handler-general/src/main/java/org/apache/shardingsphere/elasticjob/error/handler/JobErrorHandlerFactory.java

```diff
@@ -48,8 +48,8 @@
         }
         return newHandlerInstance(type);
     }
-    
```

Comment:
Keep indents (including blank lines) consistent with the previous one, please.
Ref https://shardingsphere.apache.org/community/en/contribute/code-conduct/